base/common/defines.h

```diff
@@ -70,10 +70,12 @@
 #    define NO_SANITIZE_UNDEFINED __attribute__((__no_sanitize__("undefined")))
 #    define NO_SANITIZE_ADDRESS __attribute__((__no_sanitize__("address")))
 #    define NO_SANITIZE_THREAD __attribute__((__no_sanitize__("thread")))
+#    define NO_SANITIZE_MEMORY __attribute__((__no_sanitize__("memory")))
```

Comment:
It is very difficult to use. It will work only if function reads uninitialized memory and writes nothing. I will remove to avoid confusion.